op-bindings/hardhat/types.go

```diff
@@ -19,7 +19,7 @@ type Deployment struct {
 	DeployedBytecode hexutil.Bytes      `json:"deployedBytecode"`
 	Devdoc           json.RawMessage    `json:"devdoc"`
 	Metadata         string             `json:"metadata"`
-	Receipt          Receipt            `json:"receipt"`
+	Receipt          json.RawMessage    `json:"receipt"`
```

Comment:
The serialization of the `Receipt` changed slightly, ie from `uint` to `hexutil.Uint64` and I didn't want to change things. To be honest we don't use the `Receipt` at all in Go or anywhere else that I'm aware of. This will just leave the `Receipt` as a buffer no matter what its underlying json serialization is. Can update this in the future if we do need to use the receipt